### ISSUE TITLE
[Question] Found patch size which is bigger than data

### ISSUE THREAD

**Issue by ArpanGyawali** [MIC-DKFZ/nnDetection#286]
Hi,
I am working on 3D LVO detection task and i am using nndetection. The training and everything is successful but i get this warning while training and consolidating. Does this mean something and hamper the result?
WARNING Found patch size which is bigger than data: data (6, 512, 512) patch [  8 296 296]

My data size is (number of slices, weight. height), and the segmentation is also of the same size, why does the patch size have more slices?

**Comment by partha-ghosh** [comment]
The patch size is determined based on the median shape of the dataset after it has been resampled. This ensures that the patch size captures the largest useful region across the majority of the data. Starting from there, the patch size is iteratively reduced until the memory budget is met.

That being said, the data shape of (6, 512, 512) is a bit unusal. But in this case, the patch gets padded and the network learns to deal with such scenario during training. So it does not hamper anything.

Best,
Partha

**Comment by ArpanGyawali** [comment]
Hi, @partha-ghosh , @mibaumgartner 
thats fine and everything seems good. 
But as i am training the training loss goes negative after 18-19 epochs(Total 25 epochs) and the validation loss starts increasing after very few epochs. I think this is due to overfitting.
I have around 220 training data with (80:20) train val split. That means each fold contains only 44 data. Is this because of less training data. 
Also, the training class loss starts at 1.5 and ends at only 1.2, while the train regression loss starts at negative value and gradually decrease. Why is this?

Can you give some insights to how i can increase acccuracy of my model. If more information is required i can provide.
